wordle-web/app.vue

```diff
@@ -1,15 +1,112 @@
 <template>
   <NuxtLayout>
     <v-app>
-      <v-btn @click="router.push('/')">Home</v-btn>
-      <v-btn @click="router.push('/test')">Test</v-btn>
-      <NuxtPage />
+      <v-app-bar color="primary" :elevation="2">
+
+        <template v-slot:append>
+          <v-btn icon="mdi-theme-light-dark" @click="toggleTheme" />
+          <v-btn icon="mdi-help-circle" @click="showHelpDialog = true" />
+          <HelpDialog v-model="showHelpDialog" />
+          <v-app-bar-nav-icon
+            color="secondary"
+            @click="toggleMenu"
+            :class="{ 'rotate': menuOpen}"
```

Comment:
Really cool how it rotates when the navigation drawer is open!

Reply:
Thank you, thought it looked good.